CHANGELOG.md

```diff
@@ -3,153 +3,195 @@
 ## Unreleased
 
 ### Enhancements
-  - Add OpenAPI docs for user state service
-  - Enable OpenAPI docs on all environments at /api/v1/docs
-  - Add ability to change images in pages and activities
-  - Add extrinsic user state at the resource attempt level
-  - Add bulk fetch endpoint for retrieving collection of activity attempts
-  - Add sequential page navigation to editor and full course preview
-  - Enable ecto repository stats in live dashboard
-  - Add ability to unlink a course section from an LMS
-  - Image code activity: use syntax highlighting code editor
 
 ### Bug fixes
-  - Support page-to-page links during course ingestion
-  - Use section slugs instead of ids in storage service URLs for delivery endpoints
-  - Fix a crash when an existing logged-in user accesses the Register Institution page
-  - Activity feedback fixes and unit tests
-  - Remove support for image floating to fix display issues in text editors
-  - Change activity rule, outcome modeling for use in adaptive activities
-  - Fix an issue when creating section allows multiple sections to be created
-  - Improved rendering robustness when content elements are missing key attributes
-  - Disable access to OpenAPI docs in production
+
+## 0.9.0 (2021-4-22)
```

Comment:
Am I reading this diff right? Are there two entries for 0.9.0?

Reply:
Since this PR is mostly a process step, we dont need an update changelog entry for it

Reply:
there should be 1 for unreleased and 1 for 0.9.0

Reply:
I got rid of the 0.9.1 after we talked in slack